package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "eventsource",
+  "name": "@harnessio/event",
```

Comment:
`event` is a bit too generic for this package. We should probably stick to the full `eventsource`

Reply:
Cheers fixed.